app/image/image_test.go

```diff
@@ -2,231 +2,203 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package image_test
```

Comment:
If you remove the `_test` suffix here, you must also remove the import for `"github.com/tsuru/tsuru/app/image"`. Otherwise this won't even compile.